integration-test/it-runner/src/test/java/com/example/layrry/integrationtest/LayrryIntegrationIT.java

```diff
@@ -79,6 +80,37 @@ public void runLayersFromApi() {
         assertOutput();
     }
 
+    @Test
+    public void runLayersFromApiWithFlatRepository() {
+        Layers layers = Layers.builder()
+            .layer("log")
+                .withModule("org.apache.logging.log4j:log4j-api:2.13.1")
+                .withModule("org.apache.logging.log4j:log4j-core:2.13.1")
+                .withModule("com.example.it:it-logconfig:1.0.0")
+            .layer("foo")
+                .withParent("log")
+                .withModule("com.example.it:it-greeter:1.0.0")
+                .withModule("com.example.it:it-foo:1.0.0")
+            .layer("bar")
+                .withParent("log")
+                .withModule("com.example.it:it-greeter:2.0.0")
+                .withModule("com.example.it:it-bar:1.0.0")
+            .layer("app")
+                .withParent("foo")
+                .withParent("bar")
+                .withModule("com.example.it:it-app:1.0.0")
+            .build();
+
+        layers.maven().remote()
+            .enabled(false);
+        layers.maven().local()
```

Comment:
What happens if I don't disable the remote one but add a local (flat) one?

Reply:
Both remote and local can be active at the same time. Local is always queried before remote.
This particular use case showcases a sandboxed (fully local) repository.

Reply:
Ok, makes sense.